contrib/systemd/ocid.service

```diff
@@ -9,7 +9,7 @@ EnvironmentFile=-/etc/sysconfig/ocid-storage
 EnvironmentFile=-/etc/sysconfig/ocid-network
 Environment=GOTRACEBACK=crash
 ExecStart=/usr/local/bin/ocid \
-          $OCID_STORAGE_OPTIONS \
+          $STORAGE_OPTIONS \
```

Comment:
@rhatdan Do we also need a `Require=ocid-storage-setup` to make sure ocid-storage-setup gets enabled and run before this service runs. Also Require should make sure if ocid-storage-setup fails, this service fails too.?